src/ViewLayers/ViewLayerPath.vala

```diff
@@ -119,14 +119,28 @@ public class Akira.ViewLayers.ViewLayerPath : ViewLayer {
                     context.fill ();
                 }
 
-                context.move_to (points[point_idx + 1].x + reference_point.x, points[point_idx + 1].y + reference_point.y);
```

Comment:
Something is really buggy about rendering the edit mode. I think this needs to be stabilized. Control circles disappear randomly. In some cases lines don't show up (this may be an issue with the patheditmode invariants).

Reply:
Nope. Not buggy. Just forgot to write some lines in the view layer.

Reply:
![Screenshot from 2021-12-18 13-59-04](https://user-images.githubusercontent.com/5934417/146652875-b8ff03c4-c4ad-4372-b57f-63c8d0a64770.png)

See artifacts in here. It seems the more important issues are resolved now, thanks! but these artifacts should not be hard to resolve. I'm assuming the circle radius is not being applied to the extents correctly--just a guess.